wormchain/mainnet/genesis.json

```diff
@@ -0,0 +1,351 @@
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "wormhole1cm4lqyl5j5t9ewd79vrnsrga2cnml8ca29r0sz",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        }
+      ]
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "wormhole1cm4lqyl5j5t9ewd79vrnsrga2cnml8ca29r0sz",
+          "coins": [
+            {
+              "denom": "utest",
+              "amount": "10000"
+            }
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [
+        {
+          "description": "Wormchain's native test asset",
+          "denom_units": [
+            {
+              "denom": "utest",
+              "exponent": 0,
+              "aliases": []
+            },
+            {
+              "denom": "test",
+              "exponent": 6,
+              "aliases": []
+            }
+          ],
+          "base": "utest",
+          "display": "test",
+          "name": "Test Coin",
+          "symbol": "TEST"
+        },
+        {
+          "description": "Wormchain's native staking asset",
+          "denom_units": [
+            {
+              "denom": "uworm",
+              "exponent": 0,
+              "aliases": []
+            },
+            {
+              "denom": "worm",
+              "exponent": 6,
+              "aliases": []
+            }
+          ],
+          "base": "uworm",
+          "display": "worm",
+          "name": "Worm Coin",
+          "symbol": "WORM"
+        }
+      ]
+    },
+    "capability": {
+      "index": "1",
+      "owners": []
+    },
+    "crisis": {
+      "constant_fee": {
+        "amount": "1000",
+        "denom": "worm"
+      }
+    },
+    "distribution": {
+      "delegator_starting_infos": [],
+      "delegator_withdraw_infos": [],
+      "fee_pool": {
+        "community_pool": []
+      },
+      "outstanding_rewards": [],
+      "params": {
+        "base_proposer_reward": "0.010000000000000000",
+        "bonus_proposer_reward": "0.040000000000000000",
+        "community_tax": "0.020000000000000000",
+        "withdraw_addr_enabled": true
+      },
+      "previous_proposer": "",
+      "validator_accumulated_commissions": [],
+      "validator_current_rewards": [],
+      "validator_historical_rewards": [],
+      "validator_slash_events": []
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "feegrant": {
+      "allowances": []
+    },
+    "genutil": {
+      "gen_txs": [
+        {
+          "body": {
+            "messages": [
+              {
+                "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                "description": {
+                  "moniker": "jumpcrypto",
```

Comment:
Is this "jumpcrypto" indicative of the expectation that they are / were the first guardian to stand up the chain?

Reply:
Yes we will start with one node and vote the other guardians in, allowing them to join async